tbc/items.go

```diff
@@ -207,34 +208,88 @@ func (gm GemColor) Intersects(o GemColor) bool {
 
 type ItemActivation func(*Simulation) Aura
 
-type Equipment []Item
+type Equipment [EquipTotem + 1]Item
 
-func NewEquipmentSet(names ...string) Equipment {
-	e := Equipment{EquipTotem: Item{}}
-	for _, v := range names {
-		item, ok := ItemsByName[v]
-		if !ok {
-			fmt.Printf("Unable to find item: '%s'\n", v)
-			continue
+// Structs used for looking up items/gems/enchants
+type ItemSpec struct {
+	NameOrId string
```

Comment:
I ... dont like this. Because ID is int and Name is string, having to send int types over a single string field seems kind of gross.  Thoughts on making those two fields and checking for which one is set?

Reply:
Fixed awhile ago, forgot to comment